01-CORE/07-INTEGRATIONS/bridges/secure_proxy/core/vault_manager.py

```
@@ -0,0 +1,549 @@
+        except Exception as e:
+            self.logger.error(f"Azure authentication error: {e}")
+            return None
+    
+    async def _authenticate_with_aws(self) -> Optional[str]:
+        """Authenticate to Vault using AWS IAM role."""
+        try:
+            if not self.aws_session:
+                return None
+            
+            # Get AWS credentials
+            credentials = self.aws_session.get_credentials()
```

Ruff (0.12.2)

Local variable `credentials` is assigned to but never used

Remove assignment to unused variable `credentials`

(F841)